Makefile

```diff
@@ -122,8 +123,31 @@ DISTRIBUTION_SRC=https://github.com/goharbor/distribution.git
 
 # dependency binaries
 REGISTRYURL=https://storage.googleapis.com/harbor-builds/bin/registry/release-${REGISTRYVERSION}/registry
-TRIVY_DOWNLOAD_URL=https://github.com/aquasecurity/trivy/releases/download/$(TRIVYVERSION)/trivy_$(TRIVYVERSION:v%=%)_Linux-64bit.tar.gz
-TRIVY_ADAPTER_DOWNLOAD_URL=https://github.com/goharbor/harbor-scanner-trivy/archive/refs/tags/$(TRIVYADAPTERVERSION).tar.gz
+
+ifeq ($(ARCH), arm64)
+
+
+   # Arm64-specific version and source values
+   TRIVYVERSION=v0.58.2
+   TRIVYADAPTERVERSION=v0.32.3
```

Comment:
Why are you using an older Trivy version for arm64? I think the trivy version can be the same for all architectures (The download URL would of course be different)